### PR TITLE
Add support for openSUSE tumbleweed

### PR DESCRIPTION
**Pull request by tkatsoulas**
##### Summary

TBD

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: .github/data/distros.yml [hunk 1]
```diff
@@ -162,6 +162,21 @@ include:
...
+    jsonc_removal: |
+      zypper rm -y libjson-c-devel
+    packages:
+      type: rpm
+      repo_distro: opensuse/tumbleweed
+      arches:
+        - x86_64
+        - aarch64
+    test:
+      ebpf-core: true
```
Comment (tkatsoulas) [inline .github/data/distros.yml:178]: @thiagoftsm, do you think that we need ebpf tests for `open suse tumbleweed`

Comment (thiagoftsm) [reply .github/data/distros.yml]: I think so, I remember users that had issue in the past with `OpenSUSE 15.4`.

**Review by cakrit — APPROVED** [review]
Approve docs

**Comment by tkatsoulas** [comment]
@Ferroin your workaround https://github.com/netdata/helper-images/pull/212 fixed also the issues in this effort. 

I am a little bit reluctant to merge it though until I revert  https://github.com/netdata/learn/commit/950767a3251e27c436d723763695c5a12908b1b0 (We wont produce a stable package until the next release so the kickstart will fail )

**Pull request merged**